accounts/tests.py

```diff
@@ -192,3 +194,38 @@ def test_address_display_address():
 def test_sign_out(client):
     resp = client.get("/a/sign-out")
     assertContains(resp, "signed out")
+
+
+def test_add_staff_command(db):
+    with pytest.raises(CommandError):
+        call_command("add_staff_users")
+    staff = "Name,Email,Wards\nTest Test,test1@example.org,Bad Ward\n"
+    with pytest.raises(CommandError):
+        call_command("add_staff_users", csv_file=StringIO(staff))
+
+    staff = "Name,Email,Wards\nTest Test,test2@example.org\nTester McTest,test3@example.org,Hackney Central|Victoria"
+    call_command("add_staff_users", csv_file=StringIO(staff), stdout=StringIO())
```

Comment:
This line doesn't seem to be needed due to the next line.

Reply:
Ah, this is my 100% test coverage - need at least one call with commit set to False otherwise that flow isn't followed...

Reply:
Aha! In that case maybe an assert that the number of users hasn't changed or something.